src/libs/datatableproviders/EntityServiceDataProvider.ts

```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line lodash-fp/use-fp
+import { ValueKeyIteratee } from 'lodash'
+import _ from 'lodash/fp'
+import { Ajax } from 'src/libs/ajax'
+import { DataProvider, DataProviderFeatures, DeleteTable, EntityQueryOptions, GetMetadata, GetPage } from 'src/libs/datatableproviders/DataProvider'
+
+
+export class EntityServiceDataProvider implements DataProvider {
+  constructor(namespace: string, name: string) {
+    this.namespace = namespace
+    this.name = name
+  }
+
+  namespace: string
+
+  name: string
+
+  features: DataProviderFeatures = {
+    supportsTsvDownload: true,
+    supportsTypeDeletion: true,
+    supportsTypeRenaming: true,
+    supportsExport: true,
+    supportsPointCorrection: true,
+    supportsFiltering: true
+  }
+
+  getPage: GetPage = async (signal: AbortSignal, entityType: string, queryOptions: EntityQueryOptions) => {
+    return await Ajax(signal).Workspaces.workspace(this.namespace, this.name)
+      .paginatedEntitiesOfType(entityType, _.pickBy(_.trim as ValueKeyIteratee<string | number>, {
```

Comment:
Alternatively, this cast could be avoided with something like:

```suggestion
      .paginatedEntitiesOfType(entityType, _.pickBy(v => _.trim(v?.toString()), {
```